File: 백준/Bronze/11720. 숫자의 합/숫자의 합.c
#include <stdio.h>

int main() {
	int t;
	int sum = 0;
	scanf("%d", &t);	
	char arr[t];
	scanf("%s", arr);
	for (int i = 0; i < t; i++) {
		sum += arr[i] - '0';
	}
	printf("%d", sum);


	return 0;
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int N = sc.nextInt();
        String num = sc.next();
        String arr[] = num.split("");
        int sum=0;
        for(int i=0; i<arr.length; i++) {
        	sum+=Integer.parseInt(arr[i]);
        }
        
        
        System.out.println(sum);
    }
}
